### DISCUSSION TITLE
[Feature] File Type Whitelist

### DISCUSSION THREAD

**Discussion by AngelaDMerkel** [immich-app/immich#7087] [Feature Request]
### The feature

Google and other manufacturers no longer save their DNG and JPEG images into separate folders, and instead place both images in the same directory. I do not want to back up all images to Immich, only JPEG/HEIF from my mobile, but there is currently no easy way to manage this. I have a tasker profile which moves newly taken DNG images to a different directory, but this is clumsy. 

Having the option to whitelist only certain extensions would resolve this issue. Allowing Immich to backup only JPEG and excluding DNG or MP4 would be useful. 

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#5987
